WriteHeader/calibration_tool.py

```diff
@@ -0,0 +1,105 @@
+from PIL import Image as image
+from PIL.TiffTags import TAGS
+import numpy
+import os
+import SimpleITK as sitk
```

Comment:
Try to use `itk` instead of `SimpleITK`.

Reply:
Let's make sure we only use those packages that are set up in environment.yml